### PR TITLE
Add toggles to hide the legend and the "Explore on MQE" label

### PR DESCRIPTION
**Pull request by Nathan-Fenner**
use `explore=hide`, `legend=hide` to configure the chart display. `explore` only affects the embedded view.

### CODE REVIEW

File: main/static/script.js [hunk 1]
```diff
@@ -333,16 +333,26 @@ module.controller("commonCtrl", function(
       return "rendered";
     }
   };
-  function applyDefault(name, value) {
+  $scope.hiding = {
```
Comment (achow) [inline main/static/script.js]: I'd name this hidden if we can.

**Comment by achow** [comment]
LGTM with naming nit

File: main/static/script.js (hunk 1 above)
Comment (Nathan-Fenner) [inline main/static/script.js]: Okay, I renamed it

File: main/static/script.js [hunk 2]
```diff
@@ -333,20 +333,42 @@ module.controller("commonCtrl", function(
       return "rendered";
     }
   };
-  function applyDefault(name, value) {
+  $scope.hidden = {
+    all: ($location.search().hideall || "").toLowerCase() == "true",
```
Comment (achow) [inline main/static/script.js]: How does this work?

**Comment by achow** [comment]
LGTM

**Pull request merged**